docs/api-docs/partner/becoming-a-partner.md

```diff
@@ -39,67 +40,42 @@ The following steps outline how to become a BigCommerce Partner:
 
 3. Enter your company email address.
    
-4. Click the **Technology Partner Program** button.
+4. Select the program type.
    
 ![Partner Registration](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/becoming-a-partner-02.png "Partner Registration")
 
 5. Complete the member registration.
-6. Click the **Complete Registration** button. 
-7. Once your application is approved, check your email for your Partner ID.
 
-Once approved, you will receive one or more emails listing your partner credentials and next steps. Your **Partner ID** will be required to submit your app for App Marketplace consideration. If you do not receive these credentials by email, please contact [appstore@bigcommerce.com](mailto:appstore@bigcommerce.com).
+6. Click the **Complete Registration** button. 
 
-## App Marketplace
-To build and test a Marketplace app, you will need:
+7. Once your application is approved, check your email for your partner credentials. (This can take up to two days.)
 
-* A BigCommerce store (sandbox or trial)
-* A developer account on the [BigCommerce Developer Portal](https://devtools.bigcommerce.com/my/apps)
-* BigCommerce partner credentials 
+Once approved, you will receive one or more emails listing your partner credentials and next steps. If you do not receive these credentials, please contact your BigCommerce Channel Account Manager for assistance or email [appstore@bigcommerce.com](mailto:appstore@bigcommerce.com).
 
-A sandbox store cannot process transactions and is for developing and testing apps without the 15-day time limit of a trial store. 
+## Partner Portal training
 
-We encourage you to create your sandbox store and your Developer Portal account using the same login credentials.
+BigCommerce offers training modules to help you get up to speed with the BigCommerce platform. You will need to complete these training modules to become a BigCommerce Certified Partner.
 
-> **Note**
-> Your Developer Portal account is an entirely separate account outside of the Partner Portal.
+To access the training modules, log into the [Partner Portal](https://partners.bigcommerce.com/), hover over the **Training** tab (top navigation), and select **eLearning Modules**. This will take you to the [Training & Enablement](https://partners.bigcommerce.com/English/Solutions/training/training_and_enablement.aspx) page.

```

Comment:
To access the training modules, log into the [Partner Portal](https://partners.bigcommerce.com/), hover over the **Training** tab (top navigation), and select **eLearning Modules**. This path will take you to the [Training & Enablement](https://partners.bigcommerce.com/English/Solutions/training/training_and_enablement.aspx) page.